tests/test_notifications.py

```diff
@@ -40,18 +45,39 @@ def addon():
     class AddOn:
         def __init__(self):
             self.name = "WebExtension"
-            self.paths = {
+            self._paths = {
                 "default": "webextension.xpi",
                 "corrupt": "corruptwebextension.xpi",
+                "large": "largewebextension.xpi",
             }
 
-        def get_path(self, size="default"):
-            """Returns web extension path."""
-            return self.paths.get(size, self.paths["default"])
+        @property
+        def path(self):
+            """Default path."""
+            return self._paths["default"]
+
+        def get_path(self, ext_path):
+            """Returns the web extension path."""
+            return self._paths.get(ext_path)
```

Comment:
I like where this is going but it is not the proper use of a property. `get_path` shouldn't need to exist, as we should be able to set a path and return it from the property itself. Here is a good article on what I mean, https://realpython.com/python-getter-setter/#using-properties-instead-of-getters-and-setters-the-python-way